### ISSUE TITLE
Prisma Migrate Crash (2.10.0)

### ISSUE THREAD

**Issue by nonissue** [prisma/migrate#641]
Hi Prisma Team! Prisma Migrate just crashed. 

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | darwin             | 
| Node        | v15.0.1            | 
| Prisma CLI  | 2.10.0             | 
| Binary      | af1ae11a423edfb5d24092a85be11fa77c5e499c| 

## Error
```
Error: Error in migration engine.
Reason: [libs/sql-schema-describer/src/walkers.rs:214:27] index out of bounds: the len is 0 but the index is 0

Please create an issue in the migrate repo with
your `schema.prisma` and the prisma command you tried to use 🙏:
https://github.com/prisma/migrate/issues/new

```

schema.prisma:

```
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["transactionApi"]
}

model User {
  id        Int      @id @default(autoincrement())
  createdAt DateTime @default(now()) @map(name: "created_at")
  updatedAt DateTime @default(now()) @map(name: "updated_at")
  name      String?
  role      String
  email     String?  @unique
  // posts     WeighIn[]

  @@map(name: "users")
}

model Person {
  id        Int       @id @default(autoincrement())
  createdAt DateTime  @default(now()) @map(name: "created_at")
  updatedAt DateTime  @default(now()) @map(name: "updated_at")
  name      String    @unique
  nickName  String    @unique
  weighIns  WeighIn[]
  // currentWeight Int

  @@map(name: "people")
}

model WeighIn {
  id        Int      @id @default(autoincrement())
  createdAt DateTime @default(now()) @map(name: "created_at")
  updatedAt DateTime @default(now()) @map(name: "updated_at")
  weighDate DateTime
  weight    Int

  personId Int?
  person   Person? @relation(fields: [personId], references: [id])

  @@map(name: "weighins")
}
```

**Comment by nonissue** [comment]
Might be resolved with `2.10.2`, testing now.

**Comment by nonissue** [comment]
My bad, resolved in `2.10.2`, closing.
